anypoint-signin.js

```diff
@@ -344,11 +319,19 @@ export class AnypointSignin extends ControlStateMixin(ButtonStateMixin(LitElemen
 
   constructor() {
     super();
-    this.height = 'standard';
-    this.width = 'standard';
+    /**
+     * Provide defaults that style the anypoint-signin into a button consistent
+     * with the Anypoint Components V3 button to provide consistency to the platform.
+     */
+    this.compatibility = true;
```

Comment:
I don't like this property here to be set to `true`. To set it to false it would require programmatic access to the button after it gets upgraded (meaning defined and initialized). I understand that this should be the default value in this case. Please, introduce a new property (like `material` or whatever) and when this property is set to true then in sets `compatibility` to `false`.